llpc/translator/lib/SPIRV/libSPIRV/SPIRVType.h

```diff
@@ -506,9 +506,9 @@ class SPIRVTypeStruct : public SPIRVType {
   SPIRVTypeStruct(SPIRVModule *M, SPIRVId TheId, const std::vector<SPIRVType *> &TheMemberTypes,
                   const std::string &TheName)
       : SPIRVType(M, 2 + TheMemberTypes.size(), OpTypeStruct, TheId), Literal(false) {
-    MemberTypeIdVec.resize(TheMemberTypes.size());
```

Comment:
Maybe the `resize` was intended as `reserve` (and should remain as such)?

Reply:
Did you intentionally not use a `reserve` here?

Reply:
https://github.com/GPUOpen-Drivers/llpc/pull/2348/commits/6da28f77f32803623a0542dba7ab8ecae34ba469#diff-412ad8a64497d0741782996ddb081f18b2a192924dd66996c8ba2b9485446f7a

right now it is 
"
MemberTypeIdVec.reserve(TheMemberTypes.size());
"

not resize

Reply:
Oh, my bad, I mis-interpreted the diff. Thanks!